tests/sparsezoo/utils/test_data.py

```diff
@@ -11,204 +11,115 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-from typing import Iterable
 
-import numpy
-import numpy as np
+
+from collections import OrderedDict
+
 import pytest
 
-from sparsezoo.utils import Dataset
+from sparsezoo.utils.data import DataLoader, RandomDataset
+
+
+# TESTS for DataLoader
 
 
 @pytest.fixture
-def dummy_dataset():
-    return Dataset(data=[np.random.rand(100, 10)], name="dummy-dataset")
+def dataset():
+    name, typed_shapes, num_samples = "rd1", {"inp": ([3, 224, 224], None)}, 4
+    data = RandomDataset(name=name, typed_shapes=typed_shapes, num_samples=num_samples)
+    yield data
+    del data
 
 
 @pytest.fixture
-def single_input_dataset():
-    data = [np.random.rand(3, 2), np.random.rand(3, 2), np.random.rand(3, 2)]
-    return Dataset(data=data, name="single-input-test-dataset")
+def dataset_b():
```

Comment:
small nit, but can we be more descriptive about the fixture?

Reply:
Yes sure thing!
![](https://media2.giphy.com/media/7SOdwNYHzCI7ygrLw7/giphy.gif)